packages/loot-core/src/server/budget/base.ts

```diff
@@ -95,7 +95,7 @@ function handleAccountChange(months, oldValue, newValue) {
     );
 
     months.forEach(month => {
-      let sheetName = monthUtils.sheetForMonth(month, getBudgetType());
+      let sheetName = monthUtils.sheetForMonth(month);
```

Comment:
`sheetForMonth` does not take a second parameter

Reply:
This is a bit concerning to me since there are actually two different budget types and presumably the point was to grab the appropriate one without them clobbering each other (but I haven’t seen any issues with this. @jlongster do you have any more context here?